packages/fx-core/src/component/driver/deploy/azure/azureAppServiceDeployDriver.ts

```diff
@@ -23,68 +18,37 @@ export class AzureAppServiceDeployDriver implements StepDriver {
   readonly description: string = getLocalizedString(
     "driver.deploy.deployToAzureAppServiceDescription"
   );
+  private static readonly SERVICE_NAME = "Azure App Service";
+  private static readonly SUMMARY = [
+    getLocalizedString("driver.deploy.azureAppServiceDeploySummary"),
+  ];
+  private static readonly SUMMARY_PREPARE = [
+    getLocalizedString("driver.deploy.notice.deployDryRunComplete"),
+  ];
+
   @hooks([addStartAndEndTelemetry(ACTION_NAME, TelemetryConstant.DEPLOY_COMPONENT_NAME)])
   async run(args: unknown, context: DriverContext): Promise<Result<Map<string, string>, FxError>> {
-    const impl = new AzureAppServiceDeployDriverImpl(args, context);
-    return wrapRun(
-      () => impl.run(),
-      () => impl.cleanup(),
-      context.logProvider
+    const impl = new AzureZipDeployDriverImpl(
```

Comment:
Will we rename the class in this PR or another? I would prefer to use "AzureZipDeployImpl" to distinguish the class family of V3 driver/action from Deploy interface.